balancer/rls/internal/client_test.go

```diff
@@ -69,13 +69,13 @@ func TestLookupFailure(t *testing.T) {
 	rlsClient := newRLSClient(cc, defaultDialTarget, defaultRPCTimeout)
 
 	errCh := make(chan error)
-	rlsClient.lookup("", nil, func(target, headerData string, err error) {
+	rlsClient.lookup("", nil, func(targets []string, headerData string, err error) {
 		if err == nil {
 			errCh <- errors.New("rlsClient.lookup() succeeded, should have failed")
 			return
 		}
-		if target != "" || headerData != "" {
-			errCh <- fmt.Errorf("rlsClient.lookup() = (%s, %s), should be empty strings", target, headerData)
+		if len(targets) != 0 || headerData != "" {
+			errCh <- fmt.Errorf("rlsClient.lookup() = (%v, %s), should be empty strings", targets, headerData)
```

Comment:
"should be empty strings" is no longer accurate.

Reply:
Done.